cmsplugin_filer_image/migrations/0002_delete_duplicate_thumbnailoptions.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+from django.db.models import Count, Max
+
+
+def forward(apps, schema_editor):
+    """
+    Get all the duplicates for ThumbnailOption.
+    Point each FilerImage where the duplicates are used to the correct
+    ThumbnailOption.
+    Delete remaining ThumbnailOption duplicates.
+    """
+    ThumbnailOption = apps.get_model("cmsplugin_filer_image", "ThumbnailOption")
+    unique_fields = ["name", "width", "height", "crop", "upscale"]
+    duplicates = (ThumbnailOption.objects.values(*unique_fields)
```

Comment:
The query that gets executed for each duplicate['max_id'] is the correct one, there's no need to add it to the list of values.